stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/MessagesScreen.kt

```diff
@@ -143,6 +145,22 @@ public fun MessagesScreen(
     skipEnrichUrl: Boolean = false,
     threadMessagesStart: ThreadMessagesStart = ThreadMessagesStart.BOTTOM,
     statefulStreamMediaRecorder: StatefulStreamMediaRecorder? = null,
+    topBar: (
+        @Composable (
+            Modifier,
+            MessageComposerViewModel,
+            MessageListViewModel,
+            AttachmentsPickerViewModel,
+        ) -> Unit
+    )? = null,
+    bottomBar: (
+        @Composable (
+            Modifier,
+            MessageComposerViewModel,
+            MessageListViewModel,
+            AttachmentsPickerViewModel,
+        ) -> Unit
+    )? = null,
```

Comment:
I assume customers can access the same VM instances from `Activity/Fragment`, so we could omit passing VMs into these functions.

Other than that LGTM 👍

Reply:
Hey @kanat 
This is debatable, but I get your point.
The initial implementation was to expose a `messageComposer` parameter where we needed these exact VMs.
Since we are now exposing a more generic `bottomBar` I agree that its safer to not pass the VMs.
Good catch.